packages/echo-core/src/types/legend.ts

```diff
@@ -2,3 +2,40 @@ export interface LegendOptions {
     isActive: boolean;
     selectedLegendType: string;
 }
+
+export enum LegendStatus {
```

Comment:
LegendStatus should probably not be in core. It should stay together with legend component. We want to keep most of our code in the main repo/or same repo, to make it easier to maintain and extend. If we have to code in 2-3 different repos to make a feature, we have failed. The goal of different repos is to keep the code uncoupled.

Maybe the whole legend should go into echo-components instead, I think we need to discuss this a bit. I'm not sure how coupled the legend is to other components right now, maybe we have to refactor it before we move it.

Reply:
I see! Yeah, that goal seems to be clear.
As I saw LegendStatus enum is used by Ayelix in it's own Contextmenu component, to get the legend color (which then can be passed to the context menu icon, as a background).
So it seems that it's already used without the Legend component, on it's own.

Still, if it needs to be together with the Legend component, then - for me - moving it to components would make more sense.

Checked out Legend.tsx: it doesn't seem to be coupled that much, so I think it could be moved to components...but I also don't see any indication that it should go together with `LegendStatus`.